### DISCUSSION TITLE
Original PN532 works much better!

### DISCUSSION THREAD

**Discussion by Joeknx** [rednblkx/HomeKey-ESP32#47] [General]
I only had a China clone PN532, but with it I only had a very limited range. We have a wooden door and with the chip behind the wooden frame I no longer had reception. 
I have now ordered the original from Elechouse (V4).
This means I have a significant improvement in reach and can easily get through the wooden frame.
The original PN532 is significantly more expensive, but also works significantly better.
Thank you for your suggestion to take the original!

**Comment by rednblkx** [comment]
Hi there!

Appreciate the feedback, really glad the guide was of use for you and you got this up and running.

Since you got the V4, according to Elechouse this should also give you an improvement in range of about 1~3 cm in comparison to the V3.

Anyway, shoutout to @kormax who first made this observation in their [repo](https://github.com/kormax/apple-enhanced-contactless-polling/tree/main/examples#potential-problems). 

The cheap clones are not generally bad but also none of them are as good as the original, it's a lottery but personally didn't want to bother with the shipping and taxes from Hong Kong to Europe 😅 so i ordered from a local shop which seems to be doing the job well enough so i guess i got lucky 😁.

Let me know if you have any issues or any suggestions.

**Comment by Maxime0506** [comment]
@rednblkx Can you send me a link to the original Elechouse PN532 NFC Module V3

**Reply by rednblkx** [reply]
Original ones seem to only be sold on their [website](https://www.elechouse.com/shop/) but they no longer have the V3 listed only the newer V4 which I believe is at the same price V3 was.

**Reply by Joeknx** [reply]
I also could not find a seller in germany, so I had to order it directly. The shipping is also pretty expensive.

**Reply by BuethSam** [reply]
I ordered a generic three pack from Amazon Germany and they work perfectly. It’s now how I open the door to my flat every day. But no guarantees 

https://amzn.eu/d/1SMLmd0

Edit: just saw these are V3s too. So not really related to this topic but might still be helpful for other Germans searching for modules.
